src/main/routes/test-means-allowances.ts

```diff
@@ -0,0 +1,13 @@
+import * as express from 'express'
+
+import { Paths } from 'paths'
+import * as config from 'config'
+
+/* A TEMPORARY PAGE TO TEST AZURE VAULT INTEGRATION */
```

Comment:
Can't we test without exposing vault data through an endpoint?

Reply:
i will remove before merge - but i thought ok just for preview branch so i can see it